frontend/app/routes/protected/person-case/primary-docs.tsx

```diff
@@ -268,8 +427,132 @@ function DocumentType({ currentStatus, defaultValue, errorMessage }: DocumentTyp
           required
           options={documentTypeOptions}
           label={t('protected:primary-identity-document.document-type.title')}
+          onChange={onChange}
         />
       )}
     </>
   );
 }
+
+interface PrimaryDocsFieldsProps {
+  currentStatus?: string;
+  defaultValues?: {
+    citizenshipDate: Date;
+    clientNumber: string;
+    dateOfBirth: Date;
+    gender: string;
+    givenName: string;
+    lastName: string;
+    registrationNumber: string;
+  };
+  documentType?: string;
+  errors?: Record<string, [string, ...string[]] | undefined>;
+}
+
+function PrimaryDocsFields({ currentStatus, defaultValues, errors, documentType }: PrimaryDocsFieldsProps): JSX.Element {
+  const { t } = useTranslation(handle.i18nNamespace);
+  const GENDER = ['female', 'male', 'other'] as const;
```

Comment:
This should be plural.

**Also, nitpick:** there's no reason to make it all caps. We don't follow this convention elsewhere in the app.